var/spack/repos/builtin/packages/elsi/package.py

```diff
@@ -33,27 +33,56 @@ class Elsi(CMakePackage):
     variant("enable_sips", default=False, description="Enable SLEPc-SIPs support")
     variant("use_external_elpa", default=False, description="Build ELPA using SPACK")
     variant("use_external_ntpoly", default=False, description="Build NTPoly using SPACK")
-    variant("use_external_omm", default=False, description="Use external libOMM and MatrixSwitch")
```

Comment:
`libOMM` is not available in Spack. It can be found here: https://gitlab.com/ElectronicStructureLibrary/omm/libomm

This will be tackled (eventually) in another PR.